tests-web/www/js/main.js

```diff
@@ -68,28 +60,28 @@ export class StreetExplorer {
       };
     }
 
-    // Wire up the settings button
-    const settingsButton = document.getElementById("settings");
-    settingsButton.onclick = () => {
-      if (app.settingsControl == null) {
-        app.settingsControl = makeSettingsControl(app).addTo(app.map);
-      } else {
-        app.map.removeControl(app.settingsControl);
-        app.settingsControl = null;
-      }
-    };
-
     return app;
   }
 
+  getImportSettings() {
+    try {
+      const data = new FormData(document.getElementById("import-settings"));
+      return Object.fromEntries(data);
```

Comment:
Returning the values here exactly as they come out of the form is not the most clean; the consumer has to type cast and all that. But it is less overhead to add or change settings, so I think it's worth it here.